crypto/sig_abstractions.go

```diff
@@ -25,7 +25,7 @@ import (
 
 type (
 	//ByteSignature using unspecified bound.
-	//msgp:allocbound ByteSignature
+	//msgp:allocbound ByteSignature 9999
```

Comment:
@tsachiherman do you think these need to be bounded? Any idea what will be a good bound?

Reply:
for that value, we should use the max signature size in all the Algorithmtypes we have.

Reply:
I will move this bound to a separate task/PR

Reply:
moved to https://github.com/algorand/go-algorand/pull/3196